app/services/katello/rhsm_fact_parser.rb

```diff
@@ -114,6 +115,11 @@ def get_rhsm_ip(interface)
       Net::Validations.validate_ip(ip) ? ip : nil
     end
 
+    def get_rhsm_ipv6(interface)
+      ip = facts["net.interface.#{interface}.ipv6_address.link"] || facts["net.interface.#{interface}.ipv6_address.host"]
```

Comment:
I am not sure how local link IPv6 is useful in Foreman inventory. On top of that, local link address can be also implemented with privacy extensions, meaning this address will likely often change which will cause fact changing on upload: https://datatracker.ietf.org/doc/html/rfc4941#section-2.3

Just saying. I am looking into why this does not work.

Reply:
Link local (`fe80::/8` is not useful at all). They are not routeable and require an interface specification. In Foreman we have code that drops link local addresses explicitly. See `parse_ip_address`.